posts/paired_programming_vs.md

```diff
@@ -0,0 +1,79 @@
+---
+author: Max Grover
+date: 2021-5-6
+tags: editor, paired-programming, remote-access
+---
+
+# Paired Programming using VS Code
+
+A common task when developing notebooks or packages is collaborating with others. It can be challenging to do this when using JupyterHub since two people cannot access the same notebook at same time. One solution to this is to use Visual Studio Code (VS Code) to remotely access Casper/Cheyenne, use a virtual meeting plaform (Zoom, Google Meet), and interactively collaborate on development.
+
+This process includes a few steps:
+
+1. Download VS Code
+1. Setup remote login (ssh)
+1. Enable sharing
+1. Open meeting, share the link
+1. Collaborate!
+
+## 1. Download VS Code
+
+Visual studio code is free and openly available. You can find the documentation/download information for your [specific machine here](https://code.visualstudio.com/)
+
+Once you download it, use Self Service to ensure you have the ability to install the software, then open the application once it is installed.
+
+## 2. Setup Remote Login
+
+Once you have VS Code installed, you will need to install the [Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack). These extensions are one of the aspects that make this editor so powerful.
+
+Open a terminal in VS Code after installing the extension, and type in the following (replacing `user` with your username and `machine` with `casper` or `cheyenne`):
+
+```
+ssh user@machine.ucar.edu
+```
+
+This step is to ensure that you are able to remotely access these machines.
```

Comment:
I don't think this step is needed since `Remote-SSH: Connect to Host` will take care of setting  up the SSH tunnel.

Reply:
Removed it! Thanks!